lib/V8/v8-utils.cpp

```diff
@@ -379,9 +381,10 @@ static void JS_Base64Decode(v8::FunctionCallbackInfo<v8::Value> const& args) {
 
   try {
     std::string const value = TRI_ObjectToString(isolate, args[0]);
-    std::string const base64 = StringUtils::decodeBase64(value);
+    std::string base64;
+    absl::Base64Unescape(value, &base64);
 
-    TRI_V8_RETURN_STD_STRING(base64);
+    TRI_V8_RETURN_STD_STRING(std::move(base64));
```

Comment:
Better to not use `std::move(...)` here, given that `TRI_V8_RETURN_STD_STRING(WHAT)` is a macro and refers to its `WHAT` parameter twice. The macro always doesn't benefit from using rvalues.
```suggestion
    TRI_V8_RETURN_STD_STRING(base64);
```

Reply:
Understood, thanks

Reply:
Fixed